package.json

```diff
@@ -74,7 +74,7 @@
     "rsvp": "^4.8.3"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || 10.* || >= 12.*"
```

Comment:
This also drops support for node 11, which doesn't EOL until 2019-06-01. Was that the intention?

Reply:
Yeah, but I can roll it back

Reply:
Updated to bring back Node 11 support

Reply:
2019-06-01 has passed, shall Node.js 11 support be dropped?